packages/tenant-consumer/src/tenantConsumerService.ts

```diff
@@ -0,0 +1,30 @@
+import { match } from "ts-pattern";
+import {
+  logger,
+  // consumerConfig,
+  // ReadModelRepository,
```

Comment:
I well know that this PR provide a scaffolding for services and infra, but if not relevant I suggest to remove the minor comments; maybe more convenient adding JIRA issue link that handle real consumer implementation.